VideoAPI/VideoApi.Events/Handlers/EndpointDisconnectedEventHandler.cs

```diff
@@ -0,0 +1,25 @@
+using System.Threading.Tasks;
+using VideoApi.DAL.Commands;
+using VideoApi.DAL.Commands.Core;
+using VideoApi.DAL.Queries.Core;
+using VideoApi.Domain.Enums;
+using VideoApi.Events.Handlers.Core;
+using VideoApi.Events.Models;
+
+namespace VideoApi.Events.Handlers
+{
+    public class EndpointDisconnectedEventHandler : EventHandlerBase
+    {
+        public EndpointDisconnectedEventHandler(IQueryHandler queryHandler, ICommandHandler commandHandler) : base(queryHandler, commandHandler)
+        {
+        }
+
+        public override EventType EventType => EventType.EndpointDisconnected;
+        protected override async Task PublishStatusAsync(CallbackEvent callbackEvent)
+        {
+            const EndpointState endpointState = EndpointState.Disconnected;
+            var command = new UpdateEndpointStatusCommand(SourceConference.Id, SourceEndpoint.Id, endpointState);
```

Comment:
Hi Shaed,I can't see the event being saved to the DB. We still need to do that.

Reply:
That's already handled in the controller. The previous PR handled whether or not this was an Endpoint event